docker/Dockerfile

```diff
@@ -30,8 +30,8 @@ RUN apt-get update && apt-get install -y libgeos-dev \
     && make install \
     )
 
-# Install the BCMath PHP extension.
-RUN docker-php-ext-install bcmath
+# Install the BCMath and Exif PHP extensions.
+RUN docker-php-ext-install bcmath exif
```

Comment:
Do we need to update any of our documentation regarding farmOS' expectations about its environment?

Reply:
It is a soft dependency, but good idea to at least mention it in the server requirements docs. I added a commit: https://github.com/farmOS/farmOS/pull/941/commits/dac2375568da587d7812957ff343169d85ba867b